go.mod

```diff
@@ -0,0 +1,31 @@
+module github.com/utilitywarehouse/finance-fulfilment-archive-api-cli
```

Comment:
Are we using do modules and pinning versions now? Do we need a team discussion around this?

Reply:
Probably we do need a team discussion around it.
I wanted to exercise go modules first hand, that's why I used it here.

Reply:
Know what you mean, but we should discuss it first before getting this out in the wild.